src/bentoml/_internal/bento/bento.py

```diff
@@ -123,7 +124,7 @@ class Bento(StoreItem):
 
     _info: BentoInfo
 
-    _model_store: ModelStore
+    _model_store: t.Optional[ModelStore] = None
```

Comment:
It might be needed by attrs to correctly evaluate the field types.
Not sure though, will have a test later.

Reply:
I think for Bento it is fine, because we don't serialize it to file.

We only need to use the normal types for any path that serialize to config file, because of support for <3.10

Reply:
So it is only required for cattr? Will change it.

Reply:
yes